### DISCUSSION TITLE
【冷门教程】谷歌OAuth2.0开发的正确配置步骤 | J实验室

### DISCUSSION THREAD

**Discussion by giscus** [weijunext/new-blog#6] [General]
# 【冷门教程】谷歌OAuth2.0开发的正确配置步骤 | J实验室

全栈工程师，Next.js开源手艺人，掘金签约作者，折腾才有未来

https://weijunext.com/article/integrate-google-oauth2-1

**Comment by thejoven** [comment]
重定向 URI 填写：/api/auth/callback/google
